src/packs/core_macros/Scan_3KPsYon0LoANWX54.yml

```diff
@@ -54,8 +52,13 @@ command: |-
     return sc_templates;
   }
 
-  targets.forEach((target) => {
+  targets.forEach(target => {
     let actor = target.actor;
+    if (actorPermissionLevel >= 0) {
+      const actorId = actor.system.parent._id;
+      const parentActor = game.actors.get(actorId);
+      parentActor.update({ ownership: { default: actorPermissionLevel } });
```

Comment:
Is it possible to modify only the permission on the token actor? That way unlinked tokens don't get their status permanently revealed. It's not something I've really played with, but I think that would probably be less annoying for enemies that get reused.

Reply:
I can, but that's not the use case I'm looking for with this feature. My thought on this is:
1. If the players scan a token, they know all of that token's stats, yes. However, if this token has an unlinked parent actor, they basically already know the parent's stats, too.
2. If they already know the parent's stats, they can track any children's HP/heat/stat changes by just keeping track of damage dealt/healed.
3. Therefore, by granting Observer permissions on the parent, this cuts out the busywork of taking notes and "doing the math".

My take is that if a GM who wants to enable this feature wants to truly obscure stats of different tokens, they'd use a different parent token with different stats.

Reply:
I could understand maybe removing this option from the Default scan macro, to keep that macro more laser-focused on providing a chat message Scan. The Journal Scan macro, however, is for folks who want to build up a "Pokedex" of NPCs, so those scans will provide clear statblocks that players can then use to track changes to stats like HP and Heat. In that use case, optionally providing ownership permissions at the parent actor level makes the most sense to me.